code/modules/clothing/chameleon.dm

```diff
@@ -560,10 +561,12 @@
 	if(. & EMP_PROTECT_SELF)
 		return
 	chameleon_action.emp_randomise()
+	COOLDOWN_START(src, rname_cooldown, 2 MINUTES)
 
 /obj/item/clothing/mask/chameleon/broken/Initialize(mapload)
 	. = ..()
 	chameleon_action.emp_randomise(INFINITY)
+	rname_cooldown = INFINITY  // mulligan
```

Comment:
Ironically, EMPing this will fix it, since it will get set to 2 mins instead of infinity.